src/react/components/pages/editorPage/editorPage.test.tsx

```diff
@@ -518,7 +518,31 @@ describe("Editor Page Component", () => {
 
             expect(editorPage.state().lockedTags).toEqual([]);
             expect(editorPage.state().selectedTag).toEqual(project.tags[0].name);
+        });
+
+        it("Does not set selected tag or locked tags when invalid hot key is pressed", async () => {
```

Comment:
we should have 2 tests here:

1) selecting new tag will unlock all existing locked tags
2) using a non-existent hot-key won't do anything (locked tags are still locked, no new tag selected)